esmvaltool/config-developer.yml

```diff
@@ -16,6 +16,13 @@
 #
 # Only the default drs is mandatory, the others are optional
 ###############################################################################
+PRIMAVERA:
+  input_dir: '[expid]/original_files/cmorfiles/CMIP/[institute]/[model]/[exp]/[ensemble]/[mip]/[var]/[grid]/[version]'
+  input_file: '[var]_[mip]_[model]_[exp]_[ensemble]_[grid]_*.nc'
+  output_file: '[project]_[model]_[mip]_[exp]_[ensemble]_[field]_[var]_[start_year]-[end_year]'
+  cmor_type: 'CMIP6'
+  cmor_tables: '~/PycharmProjects/earthdiagnostics/earthdiagnostics/cmor_tables/primavera'
```

Comment:
We have a set of tables like this for nearly every project that shares model data. This will be a burden to maintain and the tool will grow too large.

Reply:
Ok, I still think this needs to be addressed, but maybe not in this pull request.